packages/ui/lib/configs/guides.tsx

```diff
@@ -7,7 +8,11 @@ const GUIDES_HEADER = {
   showSocialIcons: true,
   actions: [
     {
-      content: <Button variant="outlined-primary">Launch App</Button>,
+      content: (
+        <Link href="/locks">
```

Comment:
guides is under `https://unlock-protocol.com/guides`, this will redirect to  `https://unlock-protocol.com/locks`

Reply:
How would this work when you are deploying unlock-protocol-com site to staging? Since you hardcoded the production path in get started.

Reply:
For this one there is no problem 

Redirect will be to `/locks` depending on the origin

Reply:
https://unlock-protocol.com/locks is 404...

Reply:
fixed